content/docs/add-react-to-a-website.md

```diff
@@ -81,8 +81,7 @@ React 在设计之初就可以被渐进式适配，并且**你可以根据需要
 // ... 此前粘贴的代码 ...
 
 const domContainer = document.querySelector('#like_button_container');
-const root = ReactDOM.createRoot(domContainer);
-root.render(e(LikeButton));
+ReactDOM.render(e(LikeButton), domContainer);
```

Comment:
ReactDOM.render has been deprecated from React-18.

Reply:
But in the tutorial, the link of cdn is react 17, and in the archive file, it is also react 17, is there someone who forgot to update these things